dpgen/remote/decide_machine.py

```diff
@@ -285,4 +319,4 @@ def decide_fp_machine(mdata):
 					profile['command'] = mdata['fp_command']
 					json.dump(profile, _outfile, indent = 4)
 	return mdata
-
+'''

```

Comment:
What is this?

Reply:
Comment.  I explained the reason in https://github.com/deepmodeling/dpgen/blob/d22316e71a4ae0b2cd8246b83771931e2bd778e4/dpgen/remote/decide_machine.py#L21 . The whole lines from 47 to 322 in decide_machine.py has been deprecated. No one still uses the function to automatically decide which machine to be used now.  The original codes are a mess.   So I commented all of them. We should put this function in `dpdispatcher` in a better way later in an elegant.

Reply:
I would suggest using '#' in front of all lines of the function, otherwise I cannot check which function is commented in an environment that does not have an auto highlighting like github